src/maestro/agents/scoring_agent.py

```diff
@@ -25,7 +26,6 @@ def __init__(self, agent: dict) -> None:
             self._litellm_model = raw_model
         else:
             self._litellm_model = f"ollama/{raw_model}"
-        self._opik = Opik()
 
     async def run(
```

Comment:
This method should be split into smaller sections (private) to make it readable and testable